assets/rich_text_components_definitions.ts

```diff
@@ -113,13 +118,15 @@ export default {
         "type": "custom",
         "obj_type": "SanitizedUrl"
       },
+      "default_value_obtainable_from_highlight": false,
```

Comment:
Maybe make it false by default if it's not difficult? Only add it in the components that are obtainable from highlight. It would reduce noise - we need to add this line in 8 places only for 1 true.

Reply:
Actually -- could we keep this one as-is? We generally specify everything explicitly for things like schema definitions, and it's nice to e.g. have all the keys be uniform since it makes it unambiguous what needs to be specified for a given component without needing additional context, so I'd prefer that we didn't change that general system just yet.